koji/katello-mash-split.py

```diff
@@ -315,16 +315,20 @@ def main():
     git_tag = "rpm/{}".format(branch_map[version])
 
     mash_config = "katello-{}-rhel7".format(version)
+    mash_config_pulp = "katello-{}-pulpcore-el7".format(version)
     if version == 'nightly':
         mash_config_candlepin = "katello-thirdparty-candlepin-rhel7"
     else:
         mash_config_candlepin = "katello-{}-thirdparty-candlepin-rhel7".format(version)
 
     output_paths = ["katello-{}/katello/el7".format(version)]
     output_paths_candlepin = ["katello-{}/candlepin/el7".format(version)]
+    output_paths_pulp = ["katello-{}/pulp/el7".format(version)]
```

Comment:
shouldn't this be `pulpcore`?

we used to have `pulp` in 2.x times: https://fedorapeople.org/groups/katello/releases/yum/2.4/pulp/, so maybe something new would be better? :)

Reply:
Yup, good catch